src/clusterfuzz/_internal/build_management/build_manager.py

```diff
@@ -235,6 +235,13 @@ def file_match_callback(filepath):
      with one of the |blocklisted_extensions|.
     """
     path_components = os.path.normpath(filepath).split(os.sep)
+
+    # The first component is the root directory name, which is controlled by
+    # the packaging tool. It doesn't really make sense matching against it, or
+    # otherwise we'd either match nothing, or match all files, which defeats
+    # the purpose of this callback.
```

Comment:
Not sure about this. In OSS-Fuzz we don't have a root directory. Maybe Chrome can get rid of theirs to match oss-fuzz?